cmd/tls_integration_test.go

```diff
@@ -0,0 +1,71 @@
+//go:build !windows
+// +build !windows
+
+package cmd
+
+// TODO: convert this into the integration tests, once https://github.com/grafana/k6/issues/2459 will be done
+
```

Comment:
```suggestion
```
I think we can remove this part, it is already an integration test. I think we can move it directly as a file of the `cmd/integration_tests` package.

Reply:
This is not true, sorry, my mistake.